### ISSUE TITLE
NTR: Abdominal imaging: gastric volvulus (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8990]
New term request Abdominal imaging: gastric volvulus (elthat_110711221324): Stomach; abdomen  UMLS: 0038359

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000384
- label: Gastric volvulus
- parents: [Abnormal stomach morphology (HP:0002577)](https://hpo.jax.org/app/browse/term/HP:0002577)
- pmid: PMID:29939663
- def: A gastric volvulus occurs when the stomach rotates on itself at least 180 degrees along its transverse or longitudinal axis.
- comment: Gastric strangulation is a complication that can occur after the development of acute gastric volvulus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8990
- synonyms:
